models/articles_models.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.fetchArticleByID = (id) => {
+    if(isNaN(id)){
+        return Promise.reject({status:400, msg:"Invalid article id"})
+    }
```

Comment:
You shouldn't need to manually check if id is a number, because psql will send back an error from the db.query if you try and use an invalid article id! Ideally we'd always use psql's error handling where available